README.md

```diff
@@ -42,56 +42,68 @@ yarn add @ably/asset-tracking
 
 ## Usage
 
-Here is an example of how the SDK can be used:
+### Subscribing to location updates
 
 ```ts
-import { Subscriber, Accuracy } from '@ably/asset-tracking';
+import { Subscriber } from '@ably/asset-tracking';
 
+// Client options passed to the underling ably-js instance.
+// You must provide some way for the client to authenticate with Ably.
+// In this example we're using basic authentication which means we must also provide a clientId.
+// See: https://ably.com/docs/core-features/authentication
 const ablyOptions = {
   key: ABLY_API_KEY,
   clientId: CLIENT_ID,
 };
 
+// Create a Subscriber instance.
+const subscriber = new Subscriber({
+  ablyOptions,
+});
+
+// Get an asset.
+const asset = subscriber.get('my_tracking_id');
+
 // Define a callback to be notified when a location update is recieved.
-const onLocationUpdate = (locationUpdate) => {
+asset.addLocationListener((locationUpdate) => {
   console.log(`Location update recieved. Coordinates: ${locationUpdate.location.geometry.coordinates}`);
-};
+});
+
+// Start tracking the asset. This will attach to the Ably realtime channel and enter presence.
+await subscriber.start(trackingId);
```

Comment:
Shouldn't it be `asset.start()`? 🤔 
```suggestion
await asset.start();
```

Reply:
yeah, you're right, i've fixed these now